src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    add(number) {
+      this.result += number;
+
+      return this.result;
+    },
+
+    subtract(number) {
+      this.result -= number;
+
+      return this.result;
+    },
+
+    divide(number) {
+      this.result /= number;
```

Comment:
need to add validation if number will be 0

Reply:
not fixed